### DISCUSSION TITLE
Welcome to [IrisToolbox] Discussions

### DISCUSSION THREAD

**Discussion by jaromir-benes** [IRIS-Solutions-Team/IRIS-Toolbox#291] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by djatie97** [comment]
Dear Jaromir-Benes,

Hi how are you? My name is Jati Waluyo from Central Bank of Indonesia. I hope you still remember me  :)
I have a question regarding your IRIS Toolbox.
I want to create a model for financial cycle using multivariate state space model. In this model I want to use one-sided Kalman Filter so if there are any current update to the data, the historical estimate of my financial cycle does not change.
Can we do one-sided Kalman Filter on IRIS? If yes, how to do it?
If you don't mind, could you give me your current email address, so I can contact you directly?
FYI, my email address is djatie97@gmail.com

Thank you very much.

Warm regards,

Jati
